src/components/PlayQuizComponent.tsx

```diff
@@ -1,46 +1,78 @@
 import * as React from "react";
-import { Button, Card, Dialog, Pane, Pill, TextInput } from "evergreen-ui";
-import { Quiz } from "../store/model";
+import { Button, Pane, Pill } from "evergreen-ui";
+import { Quiz, Question } from "../store/types";
 import { useState } from "react";
 import { useHistory } from "react-router";
 import { PlayersComponent } from "./PlayersComponent";
+import { ViewQuestionComponent } from "./ViewQuestionComponent";
 
 export const PlayQuizComponent = (props: {
   quiz: Quiz;
-  index: string;
+  qIndex: string;
   editPlayersMode: boolean;
 }) => {
   const quiz = props.quiz;
   const history = useHistory();
 
+  const [questionData, setQuestionData] = useState<{
+    questIndex: number;
+    catIndex: number;
+    question: Question;
+  }>({
+    questIndex: -1,
+    catIndex: -1,
+    question: null,
+  });
+
   const quizPane = (
     <Pane padding="20px">
-      {quiz.categories.map((cat, index) => (
-        <Pane key={index}>
+      {quiz.categories.map((cat, catInd) => (
+        <Pane key={catInd}>
           <h2>{cat.title}</h2>
-          {cat.questions.map((ques, i) => (
-            <Pill key={i} margin="5px" isInteractive={true}>
+          {cat.questions.map((ques, quesInd) => (
+            <Pill
+              key={quesInd}
+              margin="5px"
+              isInteractive={true}
+              onClick={() => {
+                setQuestionData({
+                  questIndex: quesInd,
+                  catIndex: catInd,
+                  question: ques,
+                });
+              }}
+            >
               {ques.points}
             </Pill>
           ))}
         </Pane>
       ))}
       <Button
         margin="5px"
-        onClick={() => history.push(`/players/${props.index}`)}
+        onClick={() => history.push(`/players/${props.qIndex}`)}
       >
         Edit Players
       </Button>
     </Pane>
   );
 
   if (props.editPlayersMode) {
-    return <PlayersComponent quiz={props.quiz} index={props.index} />;
+    return <PlayersComponent quiz={props.quiz} index={props.qIndex} />;
   }
```

Comment:
You are checking twice for whether to render PlayersComponent. You can do something like:
`const editPlayers = props.editPlayrersMode || (players.length === 0 && quiz.isPlaying)`, then do only one check. It's easier to read.